src/jit/codegenxarch.cpp

```diff
@@ -2651,9 +2638,14 @@ CodeGen::genStoreRegisterReturnInLclVar(GenTreePtr treeNode)
         getStructReturnRegisters(type0, type1, &retReg0, &retReg1);
 
         assert(retReg0 != REG_NA && retReg1 != REG_NA);
-
+        
```

Comment:
On line 2626 above we are calling genConsumeRegs(op1) where op1 is  GT_CALL.  Is genConsumeRegs() aware of calls returning structs in two registers?